composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.0",
         "fsi/datasource" : "~1.2",
-        "ruflin/elastica": "~1.2",
+        "ruflin/elastica": "~2.3",
```

Comment:
composer will not update `0.3` to `0.4` if someone has dependency from `^0.3` which should be used with `0.x` versions